workshop/controllers/ui/components/settings.lua

```diff
@@ -120,6 +120,19 @@ if shared.developerMode then
 		physicsToggle.label.text = physicsEnabled and "Stop Simulating Physics" or "Simulate Physics"
 	end)
 
+	-- Auto Save / Sync
+	local syncThread
+	local autoSaveToggle = ui.button(developmentPage, autoSave.Enabled and "Disable Auto-Save" or "Enabled Auto-Save", guiCoord(0, 190, 0, 30), guiCoord(0, 15, 0, 290), "secondary")
+	autoSaveToggle:mouseLeftPressed(function()
+		autoSave.Enabled = not autoSave.Enabled
+		if autoSave.Enabled then
+			syncThread = autoSave.Sync -- create syncThread
+		else
+			syncThread:kill() -- kill syncThread
```

Comment:
consider moving autosave logic that deals with autosave threads to the autosave module

Reply:
Again, that's on me. Will be fixed in the next commit.